src/core/manager.c

```diff
@@ -2614,15 +2614,15 @@ static int manager_dispatch_sigchld(sd_event_source *source, void *userdata) {
         return 0;
 }
 
-static void manager_start_target(Manager *m, const char *name, JobMode mode) {
-        _cleanup_(sd_bus_error_free) sd_bus_error error = SD_BUS_ERROR_NULL;
-        int r;
+static void manager_start_special(Manager *m, const char *name, JobMode mode) {
+        Job *job;
 
-        log_debug("Activating special unit %s", name);
+        if (manager_add_job_by_name_and_warn(m, JOB_START, name, mode, NULL, &job) < 0)
+                return;
 
-        r = manager_add_job_by_name(m, JOB_START, name, mode, NULL, &error, NULL);
-        if (r < 0)
-                log_error("Failed to enqueue %s job: %s", name, bus_error_message(&error, r));
```

Comment:
you drop this error message? do we already log that anyway? i don't find that so quickyl?

Reply:
manager_add_job_by_name() is replaced by manager_add_job_by_name_and_warn(), so we don't need to emit a message ourselves.